composer.json

```diff
@@ -10,6 +10,7 @@
     ],
     "require": {
         "php": "^7",
+        "doctrine/persistence": "^1.3 || ^2 || ^3",
```

Comment:
Mogelijk ook een `conflicts` met `doctrine/common` ? Of gebruiken we daar wel andere dingen uit?

Reply:
Het conflicteert niet per se met doctrine/common en ik denk dat dat al geregeld is tussen de Doctrine bundles zelf.